spec/helpers/fake_gluster.rb

```diff
@@ -178,6 +235,15 @@ def add_volumes(*volumes)
     volumes.each { |volume| add_volume(volume) }
   end
 
+  def get_volume(name)
+    (volume,) = @volumes.select { |vol| vol.name == name }
```

Comment:
We only care about the first volume? Every time I see one of these "filter/select and then take the first one" I ask something like this... I know it's usually some way to avoid nil values or something.. but it's not immediately clear. Why not use `find`?